.github/workflows/encryption.yaml

```diff
@@ -0,0 +1,103 @@
+name: Encryption
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  KIND_VERSION: v0.9.0
+  KIND_CONFIG: .github/kind-config-encryption.yaml
+
+jobs:
+  installation-and-connectivitiy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set image tag
+        id: vars
+        run: |
+          if [ ${{ github.event.pull_request.head.sha }} != "" ]; then
+            echo ::set-output name=tag::${{ github.event.pull_request.head.sha }}
+          else
+            echo ::set-output name=tag::${{ github.sha }}
+          fi
+
+      - name: Install Cilium CLI
+        run: |
+          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.4/cilium-linux-amd64.tar.gz
```

Comment:
Is this deliberately set to `v0.4` or should we consider `latest` here?

Put another way, do we want to update this regularly and how? Bump every time there's a Cilium release? Bump every time there's a cilium-cli release?

Reply:
I've removed the CI test for now from this PR as it will be more involved to get it into a useful state.